DAO.sol

```diff
@@ -552,7 +568,7 @@ contract DAO is DAOInterface, Token, TokenCreation {
 
         bool proposalCheck = true;
 
-        if (p.amount > actualBalance())
+        if (p.amount > actualBalance() || p.preSupport == false)
```

Comment:
just a small style request. I think it's cleaner to write `|| !p.preSupport)` to denote an expected false